File: app/src/main/java/com/ash/studios/musify/Utils/Instance.java
package com.ash.studios.musify.Utils;

import android.media.MediaPlayer;
import android.net.Uri;

import com.ash.studios.musify.Models.Song;

import java.util.ArrayList;

public class Instance {
    public static int position;

    public static boolean repeat;
    public static boolean shuffle;
    public static boolean playing;

    public static Uri uri;
    public static MediaPlayer mp;
    public static ArrayList<Song> songs;
}

File: app/src/main/java/com/ash/studios/musify/utils/Instance.kt
package com.ash.studios.musify.utils

import android.media.MediaPlayer
import android.net.Uri
import com.ash.studios.musify.Models.Song

object Instance {
    public var position = 0
    public var repeat = false
    public var shuffle = false
    public var playing = false
    public var uri: Uri? = null
    public var mp: MediaPlayer? = null
    public var songs: ArrayList<Song>? = null
}